docs/coding-guidelines/updating-ref-source.md

```diff
@@ -1,7 +1,7 @@
-This document provides the steps you need to take to update the reference assembly when adding new **public** APIs to an implementation assembly (post [API Review](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/adding-api-guidelines.md)).
+This document provides the steps you need to take to update the reference assembly when adding new **public** APIs to an implementation assembly (post [API Review](adding-api-guidelines.md)).
 
 ## For most assemblies within corefx
```

Comment:
Fix [MD022](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md022---headings-should-be-surrounded-by-blank-lines), and replace corefx with libraries.

```suggestion
## For most assemblies within libraries

```